### ISSUE TITLE
KA10: Game inputs

### ISSUE THREAD

**Issue by larsbrinkhoff** [rcornwell/sims#90]
I'd like to use USB joysticks and game controllers as inputs for PDP-10 games.

These changes affect at least:

- The Spacewar consoles.
- The IMX device.

**Comment by larsbrinkhoff** [comment]
I will!

**Comment by larsbrinkhoff** [comment]
I have a USB joystick providing inputs to IMX and a Spacewar console.

There needs to be some way to map joystick axes and buttons to IMX channel and console switches.

Maybe like this?  And equivalently for WCNSLS.

```
set imx axix=0:1:2    # Joystick 0, axis 1, to IMX channel 2.
set imx button=1:2:3    # Joystick 1, button 1, to IMX channel 3.
```

**Comment by larsbrinkhoff** [comment]
I'm considering having a go at this.  Never mind the current state of my branch.  I'm using SDL to access joysticks and game controllers, which I believe is OK since the display also uses SDL.  Do you have any considerations for what you think would be acceptable for merging to your "sims" repository?

**Comment by rcornwell** [comment]
The code needs to run on both Linux and Windows, (Mac OS should not be hard to port). To be merged into the main simH tree it will have to pass by Mark. If it can't be merged it needs to be in a separate directory from simH stuff so it can be managed separately. So it will need to not be part of default compile.

For doing a push request, please wait until I merge KL10 branch into main. That is where most of the PDP10 work is being done.

**Comment by larsbrinkhoff** [comment]
Resolved by #184.
